docs/migration-guides/classic-iqp-to-cloud-iqp.mdx

```diff
@@ -67,9 +67,9 @@ Jobs that were run on IBM Quantum Platform Classic will not be available after t
 
 Follow these steps to start using the new version of IBM Quantum Platform.
 
-1. If you don't already have one, [register for an IBM Cloud account.](https://cloud.ibm.com/registration?cm_sp=quantum&utm_content=quantum-next&target=https%3A%2F%2Fquantum.cloud.ibm.com&error_uri=)  If you already have an IBM Quantum Platform account, use the same email for this account.
+1. If you don't already have one, [register for an IBM Cloud account](https://quantum.cloud.ibm.com/registration). If you already have an IBM Quantum Platform account, use the same email for this account.
 2. Create an instance.
-   1. Log on to the new [IBM Quantum Platform.](https://quantum.cloud.ibm.com) Choose **IBMid** and enter your IBM Cloud account information. The dashboard opens.
+   1. Log on to the new [IBM Quantum Platform](https://quantum.cloud.ibm.com). Choose **IBMid** and enter your IBM Cloud account information. The dashboard opens.
```

Comment:
The content writers have us put punctuation inside the link

```suggestion
   1. Log on to the new [IBM Quantum Platform.](https://quantum.cloud.ibm.com) Choose **IBMid** and enter your IBM Cloud account information. The dashboard opens.
```

Reply:
In a case like this, occasionally a line break will force the punctuation onto the next line, unless the punctuation is inside the brackets.

Reply:
Here's an example (elsewhere in the docs) of the line break moving the punctuation:
<img width="749" alt="Screenshot 2025-06-19 at 9 01 38 AM" src="https://github.com/user-attachments/assets/31a78c61-c1e4-4143-85f7-73de3f2ecb8b" />

Reply:
We want the first one, with the period on the same line as the link.

Reply:
In this case:

<img width="702" alt="Screenshot 2025-06-19 at 10 24 55 AM" src="https://github.com/user-attachments/assets/8adc9da8-d1a3-4e22-9209-5347d6c2904b" />

vs

<img width="697" alt="Screenshot 2025-06-19 at 10 24 49 AM" src="https://github.com/user-attachments/assets/3f037aa7-71ea-479f-a9b9-dc2eb37d5813" />

Reply:
Correct, we want the first one.

Reply:
Reading through the Carbon documentation I just found this:
> Inline links are used in sentences or paragraphs of text. The inline link behaves the same as the standalone link but it is styled with an underline. This helps differentiate them from the text they are placed next to and makes it clear users can interact with them.
>
> Inline links should not be used on their own and **should not be paired with icons**.

Link: https://carbondesignsystem.com/components/link/usage/#inline-link